### PR TITLE
Update match-characters-that-occur-zero-or-more-times.english.md

### PR DESCRIPTION
**Pull request by ikenwaa**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Comment by ikenwaa** [comment]
After being stuck on this challenge for over 15 minutes, I went through similar challenges in the forum and found a solution that was not yet in the Github repo, I decided I should add it for Campers who could also have been trying to find a solution to the same challenge. Plus, this is my first Open Source Contribution.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md [hunk 1]
```diff
@@ -58,5 +58,9 @@ let result = chewieQuote.match(chewieRegex);
 
 ```js
 // solution required
-```
+```js
+let chewieQuote = "Aaaaaaaaaaaaaaaarrrgh!";
+let chewieRegex = /A[a]*/; // Change this line
+let result = chewieQuote.match(chewieRegex);
+console.log(result); // Aaaaaaaaaaaaaaaa
```
Comment (rupali317) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md]: console.log(result) is not required to be part of the solution as the challenge does not mention about logging the result

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md [hunk 2]
```diff
@@ -58,5 +58,9 @@ let result = chewieQuote.match(chewieRegex);
 
 ```js
 // solution required
-```
```
Comment (rupali317) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md]: actually, lines 62 to 65 should come in between line 60 and 61. Like this:

\``` js
let chewieQuote = "Aaaaaaaaaaaaaaaarrrgh!";
let chewieRegex = /A[a]*/; // Change this line
let result = chewieQuote.match(chewieRegex);
\```
We can also omit the comment: //solution required

**Review by rupali317 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md (hunk 2 above)
Comment (ikenwaa) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/match-characters-that-occur-zero-or-more-times.english.md]: Hi @rupali317 thanks for the correction. Cheers.

**Comment by RandellDawson** [comment]
Hi @ikenwaa 

It seems that similar changes have already been [accepted earlier](https://github.com/freeCodeCamp/freeCodeCamp/pull/34668) for this article you're editing, sorry about that. 😓

If you feel you have more to add, please feel free to open up a new PR.

Thanks again! 😊

---

> If you have any questions, feel free to reach out through [Gitter](https://gitter.im/FreeCodeCamp/Contributors) or by commenting below. 💬

**Pull request closed without merging**